src/main/kotlin/probation/IM.kt

```diff
@@ -0,0 +1,34 @@
+package uk.gov.justice.hmpps.architecture.probation
+
+import com.structurizr.model.Model
+import com.structurizr.model.SoftwareSystem
+import com.structurizr.view.AutomaticLayout
+import com.structurizr.view.ViewSet
+import uk.gov.justice.hmpps.architecture.HMPPSSoftwareSystem
+
+class IM private constructor() {
+  companion object : HMPPSSoftwareSystem {
+    lateinit var system: SoftwareSystem
+
+    override fun defineModelEntities(model: Model) {
+      system = model.addSoftwareSystem(
+        "IM",
+        "Interventions Manager\nHolds records of interventions delivered to services users in the community"
+      ).apply {
+        ProblemArea.GETTING_THE_RIGHT_REHABILITATION.addTo(this)
+      }
+    }
+
+    override fun defineRelationships() {
+      system.uses(Delius.system, "pushes contact information of interest to", "IAPS")
+      Delius.system.uses(system, "pushes active sentence requirements or licence conditions which are of interest to IM to", "IAPS")
```

Comment:
I think we should set a convention for where we put outbound relationships. I _think_ it makes sense to define the connections a system makes alongside the system/container (e.g. `system.uses(...)`), so I'd be tempted to move L24 into the `Delius` class.

The same goes for `src/main/kotlin/probation/EQuiP.kt` and others.

Reply:
Yeah, it's a valid discussion.

I tend to use a fuzzy "which system cares more that this happens" heuristic.

I feel the direction of 'getting data _from_ Delius' (line 24) is crucial for this system's behaviour and users. The opposite direction is more _for_ Delius (line 23).

Maybe it's something about how we phrase these relationships?

Reply:
Is it ok to defer changes to these relationships in another PR?

Reply:
Absolutely, I think we'll have a bunch of these more philosophical questions, but it shouldn't block this PR.